routes/api.js

```diff
@@ -22,6 +23,7 @@ const {
 const apiRoutes = {
   getAllBookmarks: "/bookmarks",
   getBookmarkByID: "/bookmarks/:id",
+  getBookmarkByTag: "/bookmarks/tag/:id",
```

Comment:
we can actually name the param name ```tag``` instead of ```id```
would look something like ```"/bookmarks/tag/:tag"```

Reply:
just checked the issue definition out and what @oliverwebr sort of hinted initially:

```url?tags=first,second```

looks more like a [query string](https://en.wikipedia.org/wiki/Query_string) rather than params,
the way we would get it from req looks like this:
```js
const { tags } = req.query
```

then our route string would look like:

```"/bookmarks/tags"```